src/Symfony/Bundle/FrameworkBundle/Tests/Controller/AbstractControllerTest.php

```diff
@@ -148,6 +153,11 @@ public function testGetUser()
 
     public function testGetUserAnonymousUserConvertedToNull()
     {
+        // @deprecated since Symfony 5.4
+        if (!class_exists(AnonymousToken::class)) {
+            $this->markTestSkipped('This test requires "symfony/security-core" <6.0.');
```

Comment:
This test is irrelevant in 6.0, as anonymous tokens are removed and anonymous sessions are now null anyways (so no conversion is needed in `AbstractController::getUser()`).